docker/docker-compose.dev.yml

```diff
@@ -16,7 +16,7 @@ services:
         PG_BM25_VERSION: 0.0.0
         PG_SEARCH_VERSION: 0.0.0
         PGVECTOR_VERSION: 0.5.1
-        PGML_VERSION: 2.7.10
+        PGML_VERSION: 2.7.12
```

Comment:
This is sweet! If you Ctrl/Cmd+f "2.7.10", you'll see we actually compile PGML in CI and have the version set there as well, so it needs to be modified there too. Can you confirm this version is backwards compatible + tested with ParadeDB?